doc/architecture/sensor_model.adoc

```diff
@@ -0,0 +1,6 @@
+= Sensor model
+
+Sensor models consume sensor-view messages and produce sensor-data messages.
+Those models can be based on physics or phenomenological.
```

Comment:
This does not look like a correct sentence. Maybe the paper "Towards a Generally Accepted Validation Methodology
for Sensor Models - Challenges, Metrics, and First Results" von P. Rosenberger et al. suggests a terminology. There, it says that Phenomenological is a term for both stochastic models and physical models. @PhRosenberger maybe you can comment here ;)
@kmeids

Reply:
Yes, indeed, we have a different view on those terms.
Iwould suggest to use the commonly agreed terms from the set level project:
„Those models can be reflection based (consuming rendering output from e.g. e.g. ray tracing) or object based (consuming GT object list).“

Reply:
How about that?

```suggestion
Those models can be physical models or a stochastic model.
```

Reply:
Actually, I would go for my Suggestion above, as it is clear and implementation oriented. In yours, the stochastic models are trying to mimic the physics, whereas „physical“ models could actually mean anything and nothing (like phenomenological, as well). We have ordered physical and stochastic below phenomenological in our paper, but I met a lot of people in the meantime which see it differently, because of their own specific background knowledge.

Reply:
CCB: Will delete sentence, more detail can be added later on after migration.

Reply:
sentence is removed from document